tests/features/author-delegated-grant.spec.ts

```diff
@@ -68,6 +68,127 @@ export function testAuthorDelegatedGrant(): void {
       await dwn.close();
     });
 
+    describe('ProtocolsConfigure', () => {
+      it('should allow author-delegated grant to configure protocols', async () => {
+        const alice = await TestDataGenerator.generateDidKeyPersona();
+        const bob = await TestDataGenerator.generateDidKeyPersona();
+
+        // Alice grants Bob to configure the email protocol
```

Comment:
Not seeing scoping to the email protocol, which is case in point related to earlier comment, essentially granting all power. Wondering if this is a good idea.